x-pack/plugins/infra/server/lib/log_analysis/queries/log_entry_examples.ts

```diff
@@ -32,6 +32,7 @@ export const createLogEntryExamplesQuery = (
               [timestampField]: {
                 gte: startTime,
                 lte: endTime,
+                format: 'epoch_millis',
```

Comment:
This is called when there are log rate anomalies to show in the Anomalies page, when an anomaly is expended